test/components.tsx

```diff
@@ -141,10 +133,8 @@ test('transform children', t => {
 
 test('squash multiple text nodes', t => {
 	const output = renderToString(
-		<Box unstable__transformChildren={str => `[${str}]`}>
-			<Box unstable__transformChildren={str => `{${str}}`}>
-				hello{' '}world
-			</Box>
+		<Box unstable__transformChildren={(string: string) => `[${string}]`}>
+			<Box unstable__transformChildren={(string: string) => `{${string}}`}>hello world</Box>
```

Comment:
I think it was mentioned before already, but pinging just in case, it's important to keep `hello{' '}world` as-is.

Reply:
Overly eager `xo --fix`. Stored in https://github.com/vadimdemedes/ink/pull/264/commits/92575f79e4eb69560494ecd5780c64746a19db40